src/Handler/Workspace/Create/CreateWorkspaceHandler.php

```diff
@@ -142,18 +144,32 @@ public function __invoke(
 
         $proxy->call(function () use ($cloudResourceManager, $projectName, $wsServiceAcc): void {
             $getIamPolicyRequest = new GetIamPolicyRequest();
-            $actualPolicy = $cloudResourceManager->projects->getIamPolicy($projectName, $getIamPolicyRequest, []);
+            $option = new GetPolicyOptions();
+            $option->setRequestedPolicyVersion(Helper::REQUESTED_POLICY_VERSION);
```

Comment:
Na to aby slo pracovat s conditions je potreba verzia 3 https://cloud.google.com/iam/docs/policies#versions